pycodestyle.py

```diff
@@ -122,7 +122,7 @@
 LAMBDA_REGEX = re.compile(r'\blambda\b')
 HUNK_REGEX = re.compile(r'^@@ -\d+(?:,\d+)? \+(\d+)(?:,(\d+))? @@.*$')
 STARTSWITH_DEF_REGEX = re.compile(r'^(async\s+def|def)')
```

Comment:
Not all regular expressions named similarly need to be touched all at once. This one is actually fine as is.